tests/ballerina-compiler-api-test/src/test/java/io/ballerina/semantic/api/test/SymbolLookupTest.java

```diff
@@ -393,6 +395,24 @@ public Object[][] getOnFailSymbolPositions() {
         };
     }
 
+    @Test
+    public void testSymbolLookupInModuleAlias() {
+        Project project = BCompileUtil.loadProject("test-src/symbol_lookup_with_module_alias_test.bal");
+        SemanticModel model = getDefaultModulesSemanticModel(project);
+        Document srcFile = getDocumentForSingleSource(project);
+        List<Symbol> visibleSymbols = model.visibleSymbols(srcFile, LinePosition.from(19, 25));
+        List<String> expectedModuleSymbols = List.of("obj", "sys", "regexp", "'array");
+        ArrayList<Object> symbols = new ArrayList<>();
+        for (Symbol visibleSymbol : visibleSymbols) {
+            if (visibleSymbol.kind() == MODULE
+                    && visibleSymbol.getName().isPresent()
+                    && expectedModuleSymbols.contains(visibleSymbol.getName().get())) {
+                symbols.add(visibleSymbol.getName().get());
+            }
+        }
+        assertEquals(symbols.size(), 2);
```

Comment:
Since the assertion is the count, can have a count variable, increment it in the loop and assert at the end instead

Reply:
Addressed in https://github.com/ballerina-platform/ballerina-lang/pull/38093/commits/f6fc6deae6ab6005be97a5ccea07619863dcab63.